README.md

```diff
@@ -1,3 +1,8 @@
 # Design Patterns Study Group
 
 Contains notes and exercises done as part of the Bristol DP study group.
+
+## Add maven project
+In case you open the whole repository in the IntelliJ and it doesn't give you an opportunity to run the project:
+1. Right click on a pom.xml of the particular project
```

Comment:
Right click on 'pom.xml' for the project you wish to run

Reply:
Thank you, added :)